### DISCUSSION TITLE
collaborate note -> only show notification if someone logged in and editing (should be very simple to implement).

### DISCUSSION THREAD

**Discussion by r0-n** [standardnotes/forum#2880] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
yes

**Is your feature request related to a problem? Please describe.**

_"...Because of our encrypted architecture, Standard Notes does not currently provide a real-time collaboration solution. Multiple users can share the same account however, but editing at the same time may result in sync conflicts, which may result in the duplication of notes..."_

**Describe the solution you'd like**

As a very simple "alternative", show somewhere on the screen, for example top left a small box in with the text "a user is editing a note" (or something else). This way the other user who want's to edit a note knows someone is logged in and editing a note.

For more then 2 users this might not be ideal but for 2 users (for example, my case, me and my partner) it could be more then enough.
The other user could then wait for the other user has stopped editing. You could add a timeout of X-amount of minutes/seconds that there where no changes and if that timeout is reached then the box notifying the other user should dissapear or only state a user is currently within the main standard-notes notes page if that is the case.

**Describe alternatives you've considered**
Could not find alternative.

**Additional context**
No additional context...

**Comment by myreli** [comment]
This is an interesting suggestion @r0-n, to display if/which note is currently active.
